### ISSUE TITLE
GitHub breakage on Pale Moon

### ISSUE THREAD

**Issue by CaseOf** [deu/palemoon-overlay#97]
Hello deu,
You can treat this issue when you’ll be back to work. It’s not an emergency.
GitHub is working lesser each day on Pale Moon.
For example, on my last PM update pull request, I wasn’t able to click `Create pull request` button from Pale Moon.
I had to open Firefox to do so.
Maybe would you think switching to another git host like Moonchild did for Pale Moon repo ?

**Comment by MiTereKun** [comment]
indeed, everything worked in September. An unpleasant feeling of a software product lagging behind, but I have only 2 GB of RAM on my netbook, I can't open 25 tabs at a time on any other browser to select content from social networks, only in palemoon

**Comment by jeandestouches** [comment]
Moving to another git would be better but meanwhile, this extension resolves the issues with github : 

https://github.com/JustOff/github-wc-polyfill

**Comment by MiTereKun** [comment]
Pale Moon does not open a new "io" online game [https://digdig.io](digdig.io)

and also social networks consider the browser to be outdated. Looking forward to moving to the next fully open core Gecko. Or is it the core of FF
